src/main/java/com/study/realworld/common/Func.java

```diff
@@ -0,0 +1,52 @@
+package com.study.realworld.common;
+
+import com.google.gson.JsonObject;
+import com.study.realworld.secure.SecurityFunc;
+
+import java.security.Key;
+
+public class Func {
+    public static String getErrorJson( Errors error, Object...objects ) {
+        JsonObject json = new JsonObject();
+
+        json.addProperty("E", String.valueOf(error.getCode()));
+
+        if(objects != null && objects.length > 0) {
+            for(int i = 0; i < objects.length; i = i + 2) {
+                json.addProperty(String.valueOf(objects[i]), String.valueOf(objects[i + 1]));
+            }
+        }
+        System.out.println(error.name() + " - " + error.getCode() + " - " + error.getDesc());
```

Comment:
```suggestion
        log.error("{} - {} - {}", error.name(), error.getCode(), error.getDesc());
```
에러 추적, 에러 레벨 별 디버깅 등의 이유로 println방식의 로깅은 사용을 지양하는 편이에요

[logger](https://www.baeldung.com/java-system-out-println-vs-loggers)에 대해서 한번 공부해보시는 것도 좋은 기회가 될 것 같아요 :)